pychromecast/controllers/media.py

```diff
@@ -232,6 +243,7 @@ def update(self, data):
         self.subtitle_tracks = media_data.get('tracks', self.subtitle_tracks)
         self.current_subtitle_tracks = status_data.get(
             'activeTrackIds', self.current_subtitle_tracks)
+        self.last_updated = time.time()
```

Comment:
I'm not entirely sold on the use of the `time` module here. I personally like more descriptive types than just floats/integers passed around. Maybe have a look at [`datetime.utcnow()`](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) + returning [`datetime.timedelta`](https://docs.python.org/3/library/datetime.html#timedelta-objects) as return value?